q2templates/templates/base.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset='utf-8'>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>{% block title %}{% endblock %}</title>
+    <link rel="stylesheet" href="./q2templateassets/css/bootstrap.min.css"/>
+    <style type="text/css">
+      body {
+        padding-left: 10px;
+      }
+    </style>
+    {% block head %}
+    {% endblock %}
+  </head>
+  <body>
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-lg-12">
+          <a href="http://2.qiime.org/">
+            <img class="header" src="./q2templateassets/img/qiime_logo_large.png"
```

Comment:
Can you replace `qiime_logo_large.png` with https://github.com/qiime2/logos/blob/master/raster/qiime2-rect-200.png, and remove the `width` and `height` attrs from the tag?